imperial_coldfront_plugin/templates/imperial_coldfront_plugin/invite_to_group.html

```diff
@@ -0,0 +1,11 @@
+<h1>Invitation sent to {{ invitee_email }}</h1>
```

Comment:
Similarly lets aim for something more user oriented and this should extend the base template.